.gitignore

```diff
@@ -6,3 +6,5 @@ TestServer/config/
 coverage.xml
 *~
 TestServer/storage/
+TestServer/config.dist/config/google.json
```

Comment:
In this case, the json file includes the private key for authenticating to google services. The structure is defined by google. Adding an example might cause confusion. The FCM.md describes how the json-file can be obtained.
I've added it to gitignore to prevent accidental commiting of this file

Reply:
Thanks for the clarification!